instance/migrations/0040_auto_20160420_0754.py

```diff
@@ -4,6 +4,27 @@
 from django.db import migrations, models
 
 
+def set_instance_states(apps, schema_editor):
+    SingleVMOpenEdXInstance = apps.get_model("instance", "SingleVMOpenEdXInstance")
+    for instance in SingleVMOpenEdXInstance.objects.iterator():
+        server_status = instance.server_status
```

Comment:
This is now giving `AttributeError: 'SingleVMOpenEdXInstance' object has no attribute 'server_status'`